spec/controllers/account/users_controller_spec.rb

```diff
@@ -53,6 +55,34 @@
       end
     end
 
+    context "with valid address" do
+      let!(:valid_attributes_address) { valid_params[:address_attributes] = address_valid_params }
+
+      before do
+        put :update, params: { id: user.id,
+                               user: valid_params }
```

Comment:
you don't need to write request in two lines if is not too long

Reply:
done